packages/sdk/test/integration/gc_test.ts

```
@@ -383,21 +670,45 @@ describe('Garbage Collection', function () {
     doc.update((root) => {
       delete root.list[1];
     });
+    assert.equal(
+      versionVectorHelper(doc.getVersionVector(), [
+        { actor: cli.getID()!, lamport: BigInt(3) },
+      ]),
+      true,
+    );
     assert.equal('{"list":[0,2,[3,4,5]]}', doc.toJSON());
     doc.update((root) => {
       delete (root.list[2] as Array<number>)[1];
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)